components/prism-arturo.js

```diff
@@ -0,0 +1,72 @@
+Prism.languages.arturo = {
+	'comment': {
+		pattern: /;.*/,
+		greedy: true
+	},
+
+	'character': {
+		pattern: /`.`/,
+		alias: 'char',
+		greedy: true
+	},
+
+	'string': {
+		pattern: /"(?:[^"\\\r\n]|\\.)*"/,
+		greedy: true
+	},
+
+	'regex': {
+		pattern: /\{\/.*?\/\}/,
+		greedy: true
+	},
+
+	'multistring': {
+		pattern: /"..."|».*|\{:[\s\S]*?:\}|\{[\s\S]*?\}|^-{6}$[\s\S]*/m,
+		alias: 'string',
+		greedy: true
+	},
+
+	'label': {
+		pattern: /\w+\b\??:/,
+		alias: 'class-name'
+	},
+
+	'literal': {
+		pattern: /'(?:\w+\b\??:?)/,
+		alias: 'boolean'
+	},
+
+	'type': {
+		pattern: /:(?:\w+\b\??:?)/,
+		alias: 'boolean'
+	},
+
+	'keyword1': {
```

Comment:
Right, about keyword 1 and 2. Could we find better names here? You mentioned in the docs that there are no keyword (in the sense of reserved identifiers), so what are those?

Keyword 2 seems to be built-in functions?

Reply:
basically, keyword1 are *predicate* built-in functions (all of them end with a `?` and return a logical value) and keyword2 are normal built-in functions

Reply:
Then could we call them `predicate` and `builtin-function`?

Reply:
Done ;)